### PR TITLE
Tkr doc

### LINKED ISSUE #3605: No mention of the `--tkr` option in the managed cluster documentation
## Description

There is no mention of using --tkr for creating clusters with a specific Kubernetes version in any of the managed cluster sections of the documentation.  It is briefly mentioned in the unmanaged section of the docs (https://tanzucommunityedition.io/docs/latest/ref-unmanaged-cluster/#customize-the-distribution), but that is the only mention in the documentation.

### PR DESCRIPTION
**Pull request by kcoriordan**
## What this PR does / why we need it
Adds new topic for 'Deploy Clusters with Different Kubernetes Versions'
Adds links to this new topic in:
 -managed clusters getting started guide for aws, azure, and vsphere tabs.
 -Create Workload clusters topics
I'm not sure does this also apply to Docker?

## Which issue(s) this PR fixes

Fixes: #3605

### CODE REVIEW

**Review by jpmcb — DISMISSED** [review]
Looks good to me - will let others review too

File: docs/site/content/docs/tkr-managed-cluster.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Deploy Clusters with Different Kubernetes Versions
```
Comment (joshrosso) [inline docs/site/content/docs/tkr-managed-cluster.md:1]: These instructions won't work for workload clusters.

Resolving available TKrs is all backed into the `tanzu` CLI and shouldn't be resolved using queries with `imgpkg`.

I'll send a commit to this PR updating the flow.

**Review by joshrosso — CHANGES_REQUESTED** [review]
blocking merge for incoming commit.

**Review by joshrosso — APPROVED** [review]

**Comment by joshrosso** [comment]
@jpmcb can i but you for one more review (feels like i shouldn't merge given the change is my own commit)

https://github.com/vmware-tanzu/community-edition/pull/3702/commits/304935137b88306619e747453b73a1956c09ad9b

Thank you!

**Review by jpmcb — APPROVED** [review]
LGTM!

**Pull request merged**